deps/src/singular.cpp

```diff
@@ -53,6 +75,46 @@ JLCXX_MODULE define_julia_module(jlcxx::Module & Singular)
     singular_define_ideals(Singular);
     singular_define_matrices(Singular);
 
+    Singular.method("call_interpreter", [](std::string input) {
```

Comment:
There really should be a comment here, explaining what this is all about.

Reply:
OK.